MSAL/src/MSALPublicClientApplication.m

```diff
@@ -927,6 +929,26 @@ - (void)acquireTokenWithParameters:(MSALInteractiveTokenParameters *)parameters
     {
         interactiveRequestType = MSIDInteractiveRequestLocalType;
     }
+    
+    MSIDBrokerProtocolType brokerProtocol = MSIDBrokerProtocolTypeCustomScheme;
+    MSIDRequiredBrokerType requiredBrokerType = MSIDRequiredBrokerTypeWithV2Support;
+    
+    if (@available(iOS 13.0, *))
+    {
+        requiredBrokerType = MSIDRequiredBrokerTypeDefault;
```

Comment:
Will it be better if we make it explicit to be `MSIDRequiredBrokerTypeWithNonceSupport`? :)